module/zfs/arc.c

```diff
@@ -4391,33 +4399,33 @@ __arc_shrinker_func(struct shrinker *shrink, struct shrink_control *sc)
 		return (SHRINK_STOP);
 
 	/* Reclaim in progress */
-	if (mutex_tryenter(&arc_reclaim_lock) == 0)
-		return (SHRINK_STOP);
+	if (mutex_tryenter(&arc_reclaim_lock) == 0) {
+		ARCSTAT_INCR(arcstat_need_free, ptob(sc->nr_to_scan));
+		return 0;
```

Comment:
Little things like this cause issues for my pull requests more often than I am willing to admit (although it is public record, so anyone can go look). I look forward to the repush. :)